bin/sdr

```diff
@@ -11,6 +11,21 @@ class CLI < Thor
     true
   end
 
+  no_commands do
+    def check_control_master(skip)
+      unless skip
+        say("Checking SSH connections for #{options[:environment]} environment requires a control master.")
+        control_master = ask('Have you started a control master session: (y/n): ')
+        exit unless control_master.downcase == 'y'
+      end
+
+      return if `ps aux` =~ /ControlMaster=yes/
```

Comment:
We should be able to use the `ssh` command to definitively check if the CM is present for a given named host (`sdr-infra`), and it ought to work in a cross-platform way. Like so:

```ruby
system('ssh -O check sdr-infra', err: File::NULL)
```

If ☝🏻 returns `true`, the CM is set up and running. If `false`, it's not.

Reply:
(And we think this is already working just fine given it's what we're using in dlss-capistrano: https://github.com/sul-dlss/dlss-capistrano/blob/cd4766654f600888ecb32bc3f18ff3735de9555e/lib/dlss/capistrano/tasks/control_master.rake#L23C8-L23C20)